core/src/main/scala/cats/package.scala

```diff
@@ -76,6 +76,7 @@ package object cats {
       override def get[A](fa: Id[A])(idx: Long): Option[A] =
         if (idx == 0L) Some(fa) else None
       override def isEmpty[A](fa: Id[A]): Boolean = false
+      override def iterator[A](fa: Id[A]): Iterator[A] = Some(fa).iterator
```

Comment:
Not a big deal but you can just write `Iterator(fa)`.

Reply:
I think `Iterator.single(a)` is even better because it doesn’t use varargs.